### PR TITLE
interceptors

### PR DESCRIPTION
**Pull request by AkashMT15**
i have completed interceptors with token and pass successfully .

### CODE REVIEW

File: tvminfotech-employee-UI/src/app/auth.guard.ts [hunk 1]
```diff
@@ -1,21 +1,26 @@
...
 import { ActivatedRouteSnapshot, CanActivate, Router, RouterStateSnapshot, UrlTree } from '@angular/router';
 import { Observable } from 'rxjs';
+import { SignupService } from './api/signup.service';
 
 @Injectable({
   providedIn: 'root'
 })
 export class AuthGuard implements CanActivate {
-  constructor(private routes:Router){}
+  constructor(private routes:Router ,public guarService:SignupService){}
```
Comment (Yoga2014) [inline tvminfotech-employee-UI/src/app/auth.guard.ts:10]: correct the spelling guardService

File: tvminfotech-employee-UI/src/app/login/login.component.ts [hunk 2]
```diff
@@ -64,21 +48,8 @@ navigateToNavbar(){
   }
 }
 
-signUpClick(){
-  debugger
```
Comment (Yoga2014) [inline tvminfotech-employee-UI/src/app/login/login.component.ts:68]: remove all the debugger and push the changes

File: tvminfotech-employee-services-main/src/main/resources/application.properties [hunk 3]
```diff
@@ -4,7 +4,6 @@ spring.datasource.url=jdbc:h2:mem:tvm_employee
 spring.datasource.driverClassName=org.h2.Driver
 spring.datasource.username=sa
 spring.datasource.password=c
-spring.h2.console.enabled=true
```
Comment (Yoga2014) [inline tvminfotech-employee-services-main/src/main/resources/application.properties:7]: resolve all the conflicts

**Review by Yoga2014 — CHANGES_REQUESTED** [review]

**Comment by AkashMT15** [comment]
i have resolved all the mentioned issues mam

**Pull request merged**